content/rs/clusters/monitoring/prometheus-integration.md

```diff
@@ -105,30 +104,41 @@ This is for development testing only (running in Docker).
                     - prometheus-server:prometheus
         ```
 
-    1. To start the containers, run: `docker-compose up -d`
+    1. To start the containers, run:
+
+    ```sh
+    $ docker compose up -d
+    ```
+
     1. To check that all the containers are up, run: `docker ps`
-    1. In your browser, login to Prometheus at http://localhost:9090 to make sure the server is running.
-    1. Enter **node_up** in the Expression field.
-        If Prometheus is connected to the Redis Enterprise cluster, the cluster metrics are shown.
+    1. In your browser, sign in to Prometheus at http://localhost:9090 to make sure the server is running.
+    1. Select **Status** and then **Targets** to check that Prometheus is collecting data from the Redis Enterprise cluster.
+
+        {{<image filename="images/rs/prometheus-target.png" alt="The Redis Enterprise target showing that Prometheus is connected to the Redis Enterprise Cluster.">}}{{< /image >}}
+
+        If Prometheus is connected to the cluster, you can type **node_up** in the Expression field on the Prometheus home page to see the cluster metrics.
 
 1. Configure the Grafana datasource:
-    1. Login to Grafana. If you installed Grafana locally, go to http://localhost:3000 and login with:
+    1. Sign in to Grafana. If you installed Grafana locally, go to http://localhost:3000 and sign in with:
 
         - Username: admin
         - Password: secret
 
     1. In the Grafana configuration menu, select **Data Sources**.
 
-        ![data-sources](/images/rs/data-sources.png)
+    1. Select **Add data source**.
+
+    1. Select **Prometheus** from the list of data source types.
+
+    {{<image filename="images/rs/prometheus-datasource.png" alt="The Prometheus data source in the list of data sources on Grafana.">}}{{< /image >}}
```

Comment:
The indentation of this screenshot might be causing a similar issue with the steps: `a, b, c, d, a`.
```suggestion
        {{<image filename="images/rs/prometheus-datasource.png" alt="The Prometheus data source in the list of data sources on Grafana.">}}{{< /image >}}
```